.travis.yml

```diff
@@ -3,7 +3,6 @@ sudo: required
 dist: xenial
 
 python:
-  - "2.7"
   - "3.4"
```

Comment:
remove 3.4 and 3.5 as well now. That way we can start to introduce types and the f-format strings. Some of the most helpful things in recent python versions.

Reply:
Will do! You know much I like trimming. :D